src/foam/u2/view/TableCellFormatter.js

```diff
@@ -288,6 +288,30 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
```

Comment:
Can't we just add this to the other reference formatter? Where it uses the property name if specified otherwise just uses the summary?